unity-client/Assets/Tutorial/Scripts/TutorialController.cs

```diff
@@ -39,44 +39,49 @@ internal enum TutorialPath
 
         public static TutorialController i { get; private set; }
 
-        public HUDController hudController { get => HUDController.i; }
+        public HUDController hudController
+        {
+            get => HUDController.i;
+        }
```

Comment:
Hey Brian, a doubt: These kind of indexations are made automatically by your code editor? or are made manually by you? In my opinion, in cases like this (a simple property without complex code inside) we can avoid to use 4 lines for something that can be easily read in 1. But I understand that it is something very subjective :D

Reply:
The indexing was already in place. It seems that the code formatter spread the code into multiple lines, I guess I have to review the rider C# style configuration.